src/App.tsx

```diff
@@ -1,10 +1,55 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { HeaderForm } from './components/HeaderForm/HeaderForm';
+import { Footer } from './components/Footer/Footer';
+import classNames from 'classnames';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[] | null>(null);
+  const [preparedTodos, setPreparedTodos] = useState<Todo[] | []>([]);
```

Comment:
Don't need to create 2 state for todos, use only one state and you can keep filtered todos in the variable
```suggestion
  const [todos, setTodos] = useState<Todo[]>([]);
```